numba/np/ufunc_db.py

```diff
@@ -49,7 +49,6 @@ def _fill_ufunc_db(ufunc_db):
     # module.
     from numba.np import npyfuncs
     from numba.cpython import cmathimpl, mathimpl, numbers
-    from numba.np.numpy_support import numpy_version
```

Comment:
I think this needs to stay now that it is used to check for NumPy 1.25.

Reply:
fixed in 1aef98a6bfba22d5c04615769ef03d1d9d4915ac